src/Graph/GraphFormatter.php

```diff
@@ -98,54 +98,22 @@ public function buildGraph( $nodes ) {
 		/** @var GraphNode $node */
 		foreach ( $nodes as $node ) {
 			$instance = $this;
-			$nodeLabel = htmlspecialchars( $node->getLabel() );
-
-			// take "displaytitle" as node-label if it is set
-			if ( $this->options->getNodeLabel() === GraphPrinter::NODELABEL_DISPLAYTITLE ) {
-				if ( !empty( $nodeLabel ) ) {
-					$nodeLabel = $this->getWordWrappedText( $nodeLabel, $this->options->getWordWrapLimit() );
-				}
-			}
-
-			// URL.
+			$nodeLabel = $node->getLabel();
 			$nodeLinkURL = $this->options->isGraphLink() ? "[[" . $node->getID() . "]]" : null;
+			$nodeTooltip = null;
 
 			// Display fields, if any.
 			$fields = $node->getFields();
 			if ( count( $fields ) > 0 ) {
-				$label = $nodeLabel
-					?: strtr( $this->getWordWrappedText( $node->getID(), $this->options->getWordWrapLimit() ),
-							  [ '\n' => '<br/>' ] );
-				$nodeTooltip = $nodeLabel ?: $node->getID();
-				// GraphViz is not working for version >= 1.33, so we need to use the Diagrams extension
-				// and formatting is a little different from the GraphViz extension
-				if ( version_compare( $wgVersion, '1.33', '>=' ) &&
-					\ExtensionRegistry::getInstance()->isLoaded( 'Diagrams' ) ) {
-					$nodeTooltip = str_replace( '<br />', '', $nodeTooltip );
-				}
-				// Label in HTML form enclosed with <>.
-				$nodeLabel = "<\n" . '<table border="0" cellborder="0" cellspacing="1" columns="*" rows="*">' . "\n"
-							. '<tr><td colspan="2" href="' . $nodeLinkURL . '">' . $label . "</td></tr><hr/>\n"
-							. implode( "\n", array_map( static function ( $field ) use ( $instance ) {
-								$alignment = in_array( $field['type'], [ '_num', '_qty', '_dat', '_tem' ] )
-									? 'right'
-									: 'left';
-								return '<tr><td align="left" href="[[Property:' . $field['page'] . ']]">'
-									. $field['name'] . '</td>'
-									. '<td align="' . $alignment . '">'
-										. $instance->getWordWrappedText(
-											$field['value'],
-											$instance->options->getWordWrapLimit()
-										)
-									. '</td></tr>';
-							}, $fields ) ) . "\n</table>\n>";
+				$label = $nodeLabel ?: $node->getID();
+				$nodeTooltip = htmlspecialchars( $label );
```

Comment:
The previous version replaced the '<br />' with '' when the Diagrams extension was loaded if im correct. What happened to that?

Reply:
Please elaborate. Replaced where?

Reply:
Sorry for the confusion; Github did a weird thing with my `<br />`. 

You refactored this part;
```php
if ( version_compare( $wgVersion, '1.33', '>=' ) &&
   \ExtensionRegistry::getInstance()->isLoaded( 'Diagrams' ) ) {
   $nodeTooltip = str_replace( '<br />', '', $nodeTooltip );
}
```

But I can't find that anywhere now.

Reply:
So, node tooltip for table nodes (with fields) for Diagrams should have line breaks instead of `<br />`?

Reply:
Also, a question: could you please check how Diagrams will treat `\n` escape sequence in a quoted node label or identifier, like `[label="First line\nsecond line"]` or `"First line\second line" -> 'Other node"`?